content/docs/concepts/traffic-management/index.md

```diff
@@ -678,6 +559,146 @@ spec:
 
 To see fault injection in action, see the [fault injection task](/docs/tasks/traffic-management/fault-injection/).
 
+#### Conditional rules
+
+Rules can optionally be qualified to only apply to requests that match some
+specific criteria such as the following:
+
+_1. Restrict to a specific caller_.  For example, a rule
+can indicate that it only applies to calls from workloads (pods) implementing
+the *reviews* service.
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings
+spec:
+  hosts:
+  - ratings
+  http:
+  - match:
+      sourceLabels:
+        app: reviews
+    ...
+{{< /text >}}
+
+The value of `sourceLabels` depends on the implementation of the service.
+In Kubernetes, for example, it would probably be the same labels that are used
+in the pod selector of the corresponding Kubernetes service.
+
+_2. Restrict to specific versions of the caller_. For example, the following
+rule refines the previous example to only apply to calls from version "v2"
+of the *reviews* service.
```

Comment:
These two could be collapsed into one like “restrict to specific client workloads using workload labels”. Currently it looks like we add these version labels